framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2009,6 +2013,12 @@ protected void checkTypeArguments(
         }
     }
 
+    private boolean isUninferredWildcard(AnnotatedTypeMirror type) {
```

Comment:
Maybe `isIgnoredUninferredWildcard` to highlight that it only applies if TAs are ignored.